observer.go

```diff
@@ -24,7 +24,7 @@ import opentracing "github.com/opentracing/opentracing-go"
 
 // Observer can be registered with the Tracer to receive notifications about new Spans.
 type Observer interface {
-	OnStartSpan(operationName string, options opentracing.StartSpanOptions) SpanObserver
+	OnStartSpan(sp opentracing.Span, operationName string, options opentracing.StartSpanOptions) SpanObserver
```

Comment:
@yurishkuro Agreed and thanks for the suggestion.
Have added a commit which reverts this and adds ContribObserver.